docs/release-notes/flink-1.8.md

```diff
@@ -200,7 +200,7 @@ The `CompositeSerializerSnapshot` utility class has been removed. You should
 now use `CompositeTypeSerializerSnapshot` instead, for snapshots of composite
 serializers that delegate serialization to multiple nested serializers. Please
 see
-[here](/dev/stream/state/custom_serialization.html#implementing-a-compositetypeserializersnapshot)
+[here](/projects/flink/flink-docs-release-1.8/dev/stream/state/custom_serialization.html#implementing-a-compositetypeserializersnapshot)
```

Comment:
I'm not sure about this. If we build the docs in local, this link is broken. 
I think we can use `http://ci.apache.org/projects/flink/flink-docs-release-1.8` directly here.

Reply:
Yes, it indeed does not work in local.
Fixed.